zio-http/src/test/scala/zhttp/internal/HttpRunnableSpec.scala

```diff
@@ -13,10 +13,10 @@ import zio.{Has, ZIO, ZManaged}
  * backend. For most of the other use cases directly running the HttpApp should suffice. HttpRunnableSpec spins of an
  * actual Http server and makes requests.
  */
-abstract class HttpRunnableSpec(port: Int) extends DefaultRunnableSpec { self =>
+abstract class HttpRunnableSpec(port: Int = 8088) extends DefaultRunnableSpec { self =>
```

Comment:
I think port need not be specified. Automatically a random port should be assigned for the tests.

Reply:
OR — set default port value to `0`

Reply:
Done.
Thanks for the suggestion! @tusharmath